source/deployment/sso-saml-okta.rst

```diff
@@ -47,30 +45,33 @@ Then, set **Assertion Encryption** as ``Encrypted`` and upload the **Service Pro
 	.. image:: ../../source/images/okta_6_attribute_statements.PNG
 
 8. Click **Next**. Then, set Okta support parameters for the application. Recommended settings:
- - I’m an Okta customer adding an internal app
- - This is an internal app that we have created
+ - **I’m an Okta customer adding an internal app**
+ - **This is an internal app that we have created**
 
 	.. image:: ../../source/images/okta_7_support_configuration.PNG
 
 9. Click **Finish**. On the next screen, click the **Sign On** tab and click **View Setup Instructions**.
+	9.1 Select the **Identity Provider metadata** link and copy the link from the browser URL box. This will be used during the SAML configuration steps in the next section. 
 
 	.. image:: ../../source/images/okta_8_view_instructions.PNG
 
-10. Take note of ``Identity Provider Single Sign-On URL`` (also known as ``SAML SSO URL``), and the Identity Provider Issuer, as both will be needed to configure SAML for Mattermost.
+10. Take note of **Identity Provider Single Sign-On URL** (also known as **SAML SSO URL**), and the Identity Provider Issuer, as both will be needed to configure SAML for Mattermost.
 
-Furthermore, you **must download the X.509 Public Certificate file** and save it. You will need to upload it to Mattermost at a later step.
+Furthermore, you **must download the X.509 Certificate file** and save it. You will need to upload it to Mattermost at a later step.
```

Comment:
@sbishel, will this need to be removed at a later stage and should we include a note that this isn't needed if the user is following the steps in 9.1?